### DISCUSSION TITLE
Capillary pressure values

### DISCUSSION THREAD

**Discussion by foojan-Ka** [GeoChemFoam/GeoChemFoam#13] [Q&A] [answered]
Hello,

Hope everything is fine with you.

I'm Foojan, and I have been running micromodel Test Case 02. When opening Paraview to observe the results, Pc (capillary pressure) is listed in the variables as the output value. After applying Pc variable in Paraview, there is a colormapped image showing different Pcs in pores associated with the curvature of phases interface.
I wanted to ask, for plotting the final graph of Pc vs water saturation, which value of Pc from the colormapped image should be taken in each timestep?

Looking forward to hearing from you.

Best,
Foojan

**Comment by julienmaes** [comment, marked as answer]
Hi Foojan,

Unfortunately it is no that simple. You can average pc over each phase and do the difference but it is not clear how this pressure will relate to the Darcy Pc. For one thing, the capillary number will have to be extremely small, I mean probably smaller than 1e-6. Using DNS to plot PC curve is still a matter of debates and additional research is necessary. FOr this, pore network modelling will be better

**Reply by foojan-Ka** [reply]
Hello Julien,

Many thanks for your guidance.

So, according to your answer,  I can decrease capillary number and calculate average of Pc where alpha.water is one and the other average where alpha.water is zero, then I could subtract them from each other and that is going to give me somehow the Pc in each time step and saturation. However, as you said, it is not clear how this Pc is related to Darcy Pc.
So did I understand correctly?

Best,
Foojan

**Reply by julienmaes** [reply]
Indeed. Doesn't hurt to look at it and sometimes it will be close to the Darcy one, but in my experience it's often completely different
